core/services/kraken/api/app.py

```diff
@@ -0,0 +1,54 @@
+from os import path
+
+from commonwealth.utils.apis import GenericErrorHandlingRoute
+from fastapi import FastAPI
+from fastapi.responses import RedirectResponse
+from fastapi.staticfiles import StaticFiles
+from fastapi_versioning import VersionedFastAPI
+
+# Routers
+from api.v1.routers import extension_router_v1, index_router_v1
+from api.v2.routers import (
+    container_router_v2,
+    extension_router_v2,
+    index_router_v2,
+    manifest_router_v2,
+)
+
+#
+# Main API App
+#
+
+app = FastAPI(
+    title="Kraken API",
+    description="Kraken is the BlueOS service responsible for installing and managing extensions.",
+)
+app.router.route_class = GenericErrorHandlingRoute
+
+# Adds routers to the app
+
+# API v1
+app.include_router(index_router_v1)
+app.include_router(extension_router_v1)
+
+# API v2
+app.include_router(index_router_v2)
+app.include_router(container_router_v2)
+app.include_router(extension_router_v2)
+app.include_router(manifest_router_v2)
+
+# Adds API versioning
+app = VersionedFastAPI(app, prefix_format="/v{major}.{minor}", enable_latest=True)
+
+# Adds root endpoint
```

Comment:
That following comments are almost a replicate of the code, they are unnecessary

Reply:
Removed.